package.json

```diff
@@ -17,14 +17,15 @@
   "dependencies": {
     "codemirror": "^5.22.2",
     "coffeelint": "^1.15.7",
+    "debug": "^2.6.0",
     "fbp": "~1.5.0",
     "fbp-spec": "~0.1.15",
     "flowhub-registry": "^0.1.0",
     "font-awesome": "^4.6.3",
     "indexeddbshim": "^2.2.1",
     "jshint": "^2.9.2",
     "microflo": "~0.3.43",
-    "noflo": "~0.7.9",
+    "noflo": "^0.7.9",
```

Comment:
I think side effect of trying 0.8 beta and then going back to 0.7 to get only one NoFlo into the bundle (as opposed to N pulled in via individual modules)